test/index.js

```diff
@@ -161,24 +180,6 @@ describe('big-json', function() {
         });
 
 
-        it('should throw on piping multiple source streams', function(done) {
```

Comment:
Will this result in valid behaviour?

Reply:
In retrospect, technically up to the user - if they want to parse two separate sub objects and merge them into a single object using streams, this would allow them to do so. 

This was disallowed before because the impl was done poorly. :)